### ISSUE TITLE
Kotlin+Gradle: Unresolved reference: assertThatJson

### ISSUE THREAD

**Issue by dAnjou** [lukas-krecan/JsonUnit#299]
Hi,

I'm using JsonUnit+AssertJ in a Kotlin+Gradle project and I'm getting this error:
```
$ ./gradlew test

> Task :compileTestKotlin FAILED
e: /private/tmp/DataClassKonverter/src/test/kotlin/MainKtTest.kt: (4, 40): Unresolved reference: assertThatJson
e: /private/tmp/DataClassKonverter/src/test/kotlin/MainKtTest.kt: (23, 9): Unresolved reference: assertThatJson
e: /private/tmp/DataClassKonverter/src/test/kotlin/MainKtTest.kt: (24, 13): Unresolved reference: isObject
e: /private/tmp/DataClassKonverter/src/test/kotlin/MainKtTest.kt: (25, 13): Unresolved reference: node
e: /private/tmp/DataClassKonverter/src/test/kotlin/MainKtTest.kt: (26, 13): Unresolved reference: node

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
1 actionable task: 1 executed
```
I have this in my dependencies:
```
implementation("net.javacrumbs.json-unit:json-unit-assertj:2.19.0")
```
Please also find an [example project attached](https://github.com/lukas-krecan/JsonUnit/files/5489162/example.zip).

Not sure if it's similar to #293.

**EDIT** Fixed and re-uploaded example project.

**Comment by lukas-krecan** [comment]
You need to have correct static import `import net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson`.

**Comment by dAnjou** [comment]
Sorry, there's a misunderstanding. I do want to use the Kotlin syntax, I have updated the [example project](https://github.com/lukas-krecan/JsonUnit/files/5491049/example.zip). Previously the code was Java syntax but the import was pointing to the Kotlin version just like the README is explicitly pointing out. And already the _import_ can not be resolved not just the code.

Here's the code of the test file again to prevent misunderstandings:
```kotlin
import net.javacrumbs.jsonunit.assertj.assertThatJson
import org.junit.jupiter.api.Test

internal class MainKtTest {

    @Test
    fun parseObject() {
        assertThatJson("""{"int":100}""") {
            node("int").isEqualTo(100)
        }
    }
}
```
And this is the error for this code:
```
$ ./gradlew test

> Task :compileTestKotlin FAILED
e: /private/tmp/DataClassKonverter/src/test/kotlin/MainKtTest.kt: (1, 40): Unresolved reference: assertThatJson
e: /private/tmp/DataClassKonverter/src/test/kotlin/MainKtTest.kt: (8, 9): Unresolved reference: assertThatJson
e: /private/tmp/DataClassKonverter/src/test/kotlin/MainKtTest.kt: (9, 13): Unresolved reference: node

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
1 actionable task: 1 executed
```

**Comment by lukas-krecan** [comment]
I see

**Comment by lukas-krecan** [comment]
You, are right, it does not work.

**Comment by lukas-krecan** [comment]
Fixed in 2.21.0
